### PR TITLE
Should `resolve` after fetching user.

### PR DESCRIPTION
**Pull request by dhilipsiva**
Else the promise remains unresolved.

### CODE REVIEW

File: guides/managing-current-user.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ export default Ember.Service.extend({
       if (!isEmpty(userId)) {
         return this.get('store').find('user', userId).then((user) => {
```
Comment (marcoow) [inline guides/managing-current-user.md]: this `return` can be removed as well actually

Comment (dhilipsiva) [inline guides/managing-current-user.md]: Done 👍

**Comment by marcoow** [comment]
Can you remove the [`return` here](https://github.com/simplabs/ember-simple-auth/blob/master/tests/dummy/app/services/session-account.js#L13) as well?

**Comment by dhilipsiva** [comment]
Done! 👍

**Pull request merged**